modules/wallet/consts.go

```diff
@@ -46,14 +46,9 @@ func dustValue() types.Currency {
 
 // defragFee is the miner fee paid to miners when performing a defrag
 // transaction.
-//
-// TODO: These need to be functions of the wallet that interact with the
-// transaction pool.
-func defragFee() types.Currency {
-	// 35 outputs at an estimated 250 bytes needed per output means about a 10kb
-	// total transaction, much larger than your average transaction. So you need
-	// a lot of fees.
-	return types.SiacoinPrecision.Mul64(10)
+func (w *Wallet) defragFee(txnSize uint64) types.Currency {
+	minFee, _ := w.tpool.FeeEstimation()
+	return minFee.Mul64(txnSize)
 }
```

Comment:
huh? This is still called from the tests and from `threadedDefragWallet`, for the reasons outlined in the last comment